Added option for hiding a selected option in the drop down list

Benj4m1n55 opened the pull request:
![woDHUK9fZD](https://user-images.githubusercontent.com/38720155/60022138-88bda680-9693-11e9-911c-79d2ee02890b.gif)

P.S. Hey, i currently learn programming and never did a pull request. Please let me know if i did something wrong.

brianvoe commented:
I think you did a great job! The only thing i would change is to simplify the variable just a little. You can just name it "hideSelected"

The last thing i ask from people who do pr is to add it to the docs as well. If you run "npm run dev" that will start the docs on a port shown in the terminal. There should be enough examples there that you can easliy add it to the options section(options.vue)

Thanks and let me know if you need any other help

Benj4m1n55 commented:
Thank you!
I changed the property to hideSelectedOption because i thought it's the best of both worlds. Short and clear. 😄 I also added an example to the documentation as you said. 

Again let me know if i did something wrong or something i could do better.

Best regards!
